google-cloud-core/lib/google/cloud.rb

```diff
@@ -223,9 +223,7 @@ def self.loaded_files
     # @private
     #
     def self.auto_load_files
-      if Gem.respond_to? :find_latest_files
-        return Gem.find_latest_files "google-cloud-*.rb"
-      end
+      return Gem.find_latest_files "google-cloud-*.rb" if Gem.respond_to? :find_latest_files
```

Comment:
TBH I'm not a fan of IfUnlessModifier, especially with the longer lines. I'm inclined to disable it in google-style. What do you think?

Reply:
If it will fit on one line Rubocop will fail unless it is on one line. I have just accepted this as part of the price of using a style linter. The established rules override my personal preferences.

Reply:
Agreed, but we have control over the established rules, and the authority to change them if we decide.